pkg/api/norman/customization/project/project_actions.go

```diff
@@ -41,13 +35,10 @@ type Handler struct {
 	ClusterLister            v3.ClusterLister
 	ProvisioningClusterCache provisioningcontrollerv1.ClusterCache
 	UserMgr                  user.Manager
-	PSPTemplateLister        v3.PodSecurityPolicyTemplateLister
 }
 
 func (h *Handler) Actions(actionName string, action *types.Action, apiContext *types.APIContext) error {
 	switch actionName {
-	case "setpodsecuritypolicytemplate":
-		return h.setPodSecurityPolicyTemplate(actionName, action, apiContext)
```

Comment:
Instead of dropping the case and returning the default "unrecognized action" error,  could we return an error stating that the feature is deprecated? I think it will help eliminate some confusion on both our and users' side.

Reply:
`PodSecurityPolicy` has been deprecated since version 1.25 and has now been completely removed in version 1.29. To prevent any confusion in future code, we are eliminating all references to `PodSecurityPolicy`. Retaining this portion of code could potentially lead to confusion down the line. Therefore, I believe it's best to completely remove it from the codebase.

Reply:
I realize that a dedicated error message doesn't help a lot, so your solution is good.